README.md

```diff
@@ -1,61 +1,81 @@
+# Solution
 
-# Usage
-<!-- usage -->
-```sh-session
-$ npm install -g velocitas-cli
-$ velocitas COMMAND
-running command...
-$ velocitas --version
-velocitas-cli/0.1.0 darwin-x64 node-v18.12.1
-$ velocitas --help [COMMAND]
-USAGE
-  $ velocitas COMMAND
-...
+This is where the CLI comes in - it manages everything in your repo which is not related to your _Vehicle App_'s source code as packages which are maintained by the _Velocitas_ team. The CLI allows you to download the latest versions and integrate the latest changes with ease.
+
+It also comes on a mix and match basis: You need more than the native, default runtime? Simply add the kubernetes runtime to your project. Ever want to migrate from Github to Gitee? Simply switch packages.
+
+This is enabled by 3 main features the CLI provides:
+
+* **Packages may expose programs** - These programs can be implemented in any scripting or programing language as long as the host has access to the runtime or binary of the program. The CLI masks these behind IDs. See [the exec command](#velocitas-exec-component-id) for details.
+* **Packages may synchronize files into your repository** - Unlike programs, some files cannot live outside of your checked in repository. This especially applies for Github workflows. These need to be synchronized into your repository and checked in. To do this, the CLI provides [the sync command](#velocitas-sync).
+* **Executed programs or synchronized files have access to variables** - More often than not, program configuration or places in static files need to be substituted by user-provided values. E.g. the `LOG_LEVEL` of a runtime service or the `REPO_URL` for links in Markdown files. Therefore the CLI parses all static files it synchronizes and substitutes all occurrences of values that match the following expression and are a defined variable: `${{ variableName }}`. For invoked programs, all defined variables are exposed as ENV variables for the program to use. See the [variables feature](./docs/features/VARIABLES.md) for details.
+
+# Getting started
+
+The CLI manages packages - each of which is simply a git repository with a `manifest.json` at its root. There is no central package registry. Any git repository can be referenced as a package. The ones currently maintained by the Eclipse Velocitas team can be found [here](https://github.com/eclipse-velocitas/?q=devenv).
+
+When starting a new project from scratch, use the command `velocitas init`. This command will create a fresh `.velocitas.json` inside the current working directory with a few default packages already referenced and download. Feel free to adjust these packages to suite your needs.
+
+If you use one of our [template repositories](https://github.com/eclipse-velocitas/?type=template) to kick-start your project, your project already comes with a `.velocitas.json`. Check out the repo and execute `velocitas init` in your new repository. This will download and initialize all referenced packages.
```

Comment:
Should we maybe explain the .velocitas.json also in the cli readme, since this is the link between repos + packages?

Reply:
You are right, will add that.